framework/helpers/logger/README.md

```diff
@@ -86,6 +89,19 @@ loggerSetTarget the long id of field "Log" of me
 
 You can check where log messages are being sent using the `loggerGetTarget()` function.
 
+
+
+## Configuring app.yml
+
+Logger can be configured via the following keys in your `app.yml` file.
+
+|  Type  |  Description  |
+|------------|---------------|
+| `logger-types` | Calls [loggerSetTypes](#loggerSetTypes) to set the types of messages to log |
+| `logger-target` | Calls [loggerSetTarget](#loggerSetTarget) to specify where your log messages should go |
+| `logger-include-log-type` | Calls [loggerSetIncludeLogType](#loggerSetIncludeLogType) to include (or exclude) the log type from the message |
+
+
```

Comment:
Reverse order:
yes, I agree, they should be nested.  I have several other things that I want to add as well, like the delimiters (future version, perhaps even on this PR), and it would be cleaner if they were nested.

Remember, this is a helper.  There is an API to it.  So now, everything is set via the API - you have to stick in extra code to make the settings.  Since some of these things might not change, putting it in the app.yml file gives the developer to set them there and forget about them.

I have multiple cases, on apps that are running as servers, where we are firing logs constantly.  Actually, in the one app, we have two logs:  one for normal operations, and one for exceptions/errors.

Having the logging settings pre-configured is also useful if you want to arm the logger but not turn it on.  For example, you can have logging settings built in, but then not begin logging unless the user selects a menu item.  Yes, you could have all of that hard-coded.  This way you wouldn't have to.

I would also contend that you should think about levure as more than just a framework for compiling/building apps.  I would be willing to bet that there are more uncompiled LC apps running than compiled.  In that situation, app.yml becomes a way to adjust settings in a central place.

Reply:
Thanks for the info on configuration. As you add delimiters and such it does make more sense. 

As for how I think of Levure, I’m not sure what you mean. It doesn’t really change how one thinks of app.yml. It is a project file regardless.

Reply:
What I'm saying is that if I or some other developer wants to modify certain behaviors/parameters, the app.yml file is a convenient place to do that.

Reply:
Well I agree whole heartedly with that. I have fairly large `app.yml` files in my projects :-)

My question as to whether or not configuration for logger should happen in `app.yml` isn't about compiled or not. I have many helpers that use `app.yml`. I just want to make sure there are practical use cases before adding it.

Reply:
For me that answer would be "yes" because then I can turn it on or off or set the target or the other parameters without having to do it in the code.  The only other way to get this functionality would be with a separate prefs file.  It's your baby, so you have to decide if you're ok with it or not.  I'd rather avoid the prefs file and keep it all central.

Reply:
I'm okay with using `app.yml` given the types of properties you want to add.